internal/cephfs/volumemounter.go

```diff
@@ -176,7 +177,8 @@ func mountFuse(ctx context.Context, mountPoint string, cr *util.Credentials, vol
 	// and PID of the ceph-fuse daemon for unmount
 
 	match := fusePidRx.FindSubmatch(stderr)
-	if len(match) != 2 {
+	const reqLenForPID = 2
```

Comment:
This variable is badly named, I think. `match` should contain two items, it is not really the "length for the PID", as the PID can be any number between 1-4194304. The PID gets stored in `match[1]`, and `starting fuse` is probably in `match[0]`